src/pages/settings/InitialSettingsPage.js

```diff
@@ -259,9 +268,7 @@ function InitialSettingsPage(props) {
                                 ref={popoverAnchor}
                                 hoverAndPressStyle={styles.hoveredComponentBG}
                                 shouldBlockSelection={Boolean(item.link)}
-                                onSecondaryInteraction={
-                                    !_.isEmpty(item.link) ? (e) => ReportActionContextMenu.showContextMenu(CONST.CONTEXT_MENU_TYPES.LINK, e, item.link, popoverAnchor.current) : undefined
-                                }
+                                onSecondaryInteraction={item.link ? (event) => openPopover(item.link, event) : undefined}
```

Comment:
coming from - https://github.com/Expensify/App/issues/60783

this caused a minor bug related to longpress and tooltips